autoencoder/config.py

```diff
@@ -0,0 +1,29 @@
+import ml_collections
+from transformers import AutoConfig
+
+def config():
+    config = ml_collections.ConfigDict()
+    encoder = config.encoder = ml_collections.ConfigDict()
+    encoder.name = "bert-base-uncased"
+    encoder.config = AutoConfig.from_pretrained(encoder.name)
+    
+    compressor = config.compressor = ml_collections.ConfigDict()
+    compressor.num_latents = 32
+    compressor.embedding_dim = 768
+    compressor.hidden_size = 768
+    compressor.n_heads = 12
+    compressor.n_layers = 4
```

Comment:
LD4TG utilizes 3 layers for both autoencoding modules, I think for consistency purposes it is wise to follow their implementation